buildout.cfg

```diff
@@ -26,14 +27,10 @@ eggs = z3c.checkversions [buildout]
 recipe = plone.recipe.codeanalysis[recommended]
 directory = ${buildout:directory}/src/collective/cover
 clean-lines = True
-clean-lines-exclude = ${buildout:directory}/src/collective/cover/static/js/vendor/
-csslint = True
-csslint-bin = bin/csslint
+clean-lines-exclude = ${:directory}/browser/static
 debug-statements = True
 flake8-ignore = B901,D001,E501,P001,T000
 flake8-max-complexity = 12
-jshint = True
```

Comment:
Can't check be done in the webpack folder?

Reply:
IMO, it doesn't make sense to run the linters on code generated by webpack.